programs/relayer/src/modules/cosmos_to_eth.rs

```diff
@@ -100,7 +99,7 @@ impl RelayerService for CosmosToEthRelayerModuleServer {
                     .await
                     .map_err(|e| tonic::Status::from_error(e.to_string().into()))?,
                 ibc_version: "2".to_string(),
```

Comment:
Should this also be like an enum or number or something? Feels a bit strange to use a string for this

Reply:
I think this interface isn't finalized enough yet to warrant an enum. Might even fully remove it.